dotcom-rendering/src/web/server/pageTemplate.ts

```diff
@@ -320,8 +320,24 @@ https://workforus.theguardian.com/careers/product-engineering/
 					window.curl = window.curlConfig;
 				</script>
 
-				${bork ? `<script>(${fid.toString()})()</script>` : ''}
-				${bork ? `<script>(${fcp.toString()})()</script>` : ''}
+
+const mvtId = Number(
```

Comment:
Nice borking :)

Would this be a cookie read of `GU_mvt_id` for every page load whether borking was active or not? Could it be moved inside the script when `borkWebVitals === true`? Also I might have missed something but I don't see `mvtId` used anywhere?

Reply:
This was a copy pasta. Gone